### ISSUE TITLE
[question] Is it possible to upload source code archive to artifactory, while using a conan-community recipe

### ISSUE THREAD

**Issue by AndreyMlashkin** [conan-io/conan-center-index#18999]
### What is your question?

All conan-center-index recipes are relying on github/gitlab or other repository to have a link with the sources.
Sometimes, the service may be down or the repository may be not available anymore.
Is is possible to store sources archive in artifactory in order not to be relied upon the original sources provider?

**Comment by whaliendev** [comment]
Indeed, we can setup a conan remote locally with [setting up conan remotes](https://docs.conan.io/2/tutorial/conan_repositories/setting_up_conan_remotes.html).
We internally host a JFrog Artifactory

**Comment by AndreyMlashkin** [comment]
If my question was not clear enogh, I will reprhrase it with an example:
conan_export.tgz contains conandata.yml
conan_sources.tgz contains patches

Actual sources archive comes from a download link from conandata.yml, for zlib it's
https://github.com/madler/zlib/releases/download/...

Some of the source code sources are not reliable. Is it possible to store actual sources code archive on artifactory or even local conan server?

@madebr
